homeassistant/components/blueprint/websocket_api.py

```diff
@@ -36,12 +42,8 @@ async def ws_list_blueprints(hass, connection, msg):
     )
     results = {}
 
-    for domain, domain_results in zip(
-        domain_blueprints,
-        await asyncio.gather(
-            *[db.async_get_blueprints() for db in domain_blueprints.values()]
-        ),
-    ):
+    def get_domain_blueprint(domain, domain_results):
```

Comment:
What's this "getting". It feels more like `process_domain_blueprints` ? It should also be decorated with `@callback`. But if we go to support just 1 domain, we can just inline it.